subsys/bluetooth/host/hci_core.c

```diff
@@ -2807,20 +2830,16 @@ static void hci_event(struct net_buf *buf)
 	net_buf_unref(buf);
 }
 
-static void send_cmd(void)
+static void _send_cmd(void)
```

Comment:
What's the purpose of adding `_` in front of a `static` function? Double private?

Reply:
🕵️double-private.
I'll remove it :)

In general there might be some weird stuff like this that I missed and also some meme-ing that I forgot to remove <sub>sorry about that</sub>. Thanks for the proof-reading

Reply:
We need a boomer-check in CI to catch these :D